### ISSUE TITLE
Installer and restart option (Bugzilla Bug 1602)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#1602]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 1602
Date: 2016-04-07 15:57:03 +0000
From: Francisco Panis Kaseker &lt;<apterix@gmail.com>&gt;
To:   OpenMandriva QA &lt;<bugs@openmandriva.org>&gt;
CC:   gio_6b@yahoo.it, @itchka, @tapwag, @rugyada, @tpgxyz

Last updated: 2018-09-05 22:16:42 +0000

## Comment 12640
Date: 2016-04-07 15:57:03 +0000
From: Francisco Panis Kaseker &lt;<apterix@gmail.com>&gt;

When we finish installation we see "restart" option. If I do not mark this option and click in quit/exit, install will reboot ignoring that.

## Comment 12659
Date: 2016-04-07 18:02:29 +0000
From: @tapwag

Is upstream aware of this or is it our implementation of the installer?

Setting to NEEDINFO.

## Comment 12682
Date: 2016-04-08 09:53:36 +0000
From: @rugyada

I do not see any difference whether or not "Restart now" is ticked or unticked.
Francisco could you please write on your bug reports which ISO you are testing? It may help, thank you.

## Comment 12691
Date: 2016-04-08 16:30:36 +0000
From: Francisco Panis Kaseker &lt;<apterix@gmail.com>&gt;

OpenMandrivaLx_3beta1.x86_64.iso
md5 deadb368ab9a549f94da7c1f6f8ada93

## Comment 12692
Date: 2016-04-08 16:53:07 +0000
From: @rugyada

(In reply to Francisco Panis Kaseker from comment #3)
> OpenMandrivaLx_3beta1.x86_64.iso
> md5 deadb368ab9a549f94da7c1f6f8ada93

Thank you. Just to be on the safe side about what we're speaking of :)

Once install is finished, actually one should not restart because if the system finds the ISO or DVD it wants to launch installation once more.
So one should better: shutdown, eject/unmount support, and boot again instead of "reboot".

## Comment 12715
Date: 2016-04-09 10:36:58 +0000
From: @tapwag

So I am assuming this is a question of labeling the checkboxes and buttons. Did we at OpenMandriva make any changes to this? 

I have an account with the Calamares people and can certainly file a suggestion.

## Comment 12752
Date: 2016-04-10 11:14:56 +0000
From: @rugyada

I have the feeling that "Quit" should shutdown the system instead of reboot because in my experience I find that atm both are giving the same result.
So imho could be a calamares bug/issue.

## Comment 12753
Date: 2016-04-10 11:21:59 +0000
From: @rugyada

(In reply to rugyada from comment #6)
> I have the feeling that "Quit" should shutdown the system instead of reboot
> because in my experience I find that atm both are giving the same result.
> So imho could be a calamares bug/issue.

Nevertheless, there is another fact to consider: we can run calamares install both by live mode and by the ISO boot menu.
When we run it by live mode , the "Quit" button exit the install and let us play in live mode. So it's not really correct either.

## Comment 12754
Date: 2016-04-10 11:29:34 +0000
From: @rugyada

In short, calamares *could* miss a shutdown option which works the same way regardless: quit/exit/leave/shutdown/turn-off/close/stooop! :)

## Comment 12853
Date: 2016-04-16 09:44:49 +0000
From: @rugyada

Waiting for fixing or upstream feature, if any.

While install is run from ISO main menu:
Quit > 
Main menu > Other > Poweroff

Eject/unmount your support, boot your fresh system.

## Comment 12941
Date: 2016-04-20 16:30:03 +0000
From: @cris-b

*** Bug #1446 has been marked as a duplicate of this bug. ***

## Comment 12952
Date: 2016-04-20 21:56:14 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

(In reply to crisb from comment #10)
> *** Bug #1446 has been marked as a duplicate of this bug. ***

Don't agree, see bug #1446.

## Comment 13328
Date: 2016-05-31 16:03:11 +0000
From: @tpgxyz

Feel free to provide a patches to calamares installer which will do:

1. recognize if executed form CD/DVD media
2. eject occupied CD/DVD tray

Thanks.

## Comment 17934
Date: 2018-03-05 19:18:02 +0000
From: @itchka

Most of this has been addressed in the later versions of calamares
